### PR TITLE
Feature ge appointment scheduling

### PR DESCRIPTION
**Pull request by denisfruza**
Secretary can choose one of the appointment search results and schedule an examination.
After choosing appointment, he selects patient and then schedules an examination
There is a button "Schedule" for each search result.

### CODE REVIEW

File: WPFHospitalEditor/ScheduleWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        </Style>
+    </Window.Resources>
+    <Grid>
+        <Border BorderThickness="10" BorderBrush="#FF8BD9E0" x:Name="Border">
+            <Grid x:Name="DynamicGrid" Background="#FFC6F5F8">
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Top" Margin="0,10,0,0">
+                    <ComboBox Name="patients" Width="200"/>
+                </StackPanel>
+                <StackPanel Orientation="Vertical" HorizontalAlignment="Center" VerticalAlignment="Bottom">
+                <Label x:Name="appointmentInfo" Content="dick" HorizontalAlignment="Center" VerticalAlignment="Bottom"></Label>
```
Comment (BoKna031) [inline WPFHospitalEditor/ScheduleWindow.xaml:26]: change content of label

File: WPFHospitalEditor/ScheduleWindow.xaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Bottom">
```
Comment (rammba) [inline WPFHospitalEditor/ScheduleWindow.xaml:27]: image

File: WPFHospitalEditor/ScheduleWindow.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            {
+                MessageBox.Show("You must select a patient!");
+            }
+            else { 
+                String patient = patients.SelectedItem.ToString();
+                int patientID = int.Parse(patient.Split(" ")[0]);
+                if (examinationServerController.ScheduleExamination(recommendationDto.TimeInterval.Start, recommendationDto.Doctor.Id, patientID).Equals("OK"))
+                {
+                    MessageBox.Show("Examination has been scheduled succesfuly!");
+                }
```
Comment (rammba) [inline WPFHospitalEditor/ScheduleWindow.xaml.cs:49]: Typo -> successfully :)

File: WPFHospitalEditor/Controller/Interface/IExaminationServerController.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace WPFHospitalEditor.Controller.Interface
+{
+    public interface IExaminationServerController
+    {
+        public string ScheduleExamination(DateTime startTime, int DoctorId, int patientId);
+    }
```
Comment (rammba) [inline WPFHospitalEditor/Controller/Interface/IExaminationServerController.cs:8]: Public access modifier here is not needed, same mistake in IExaminationServerService.cs

File: WPFHospitalEditor/ScheduleWindow.xaml (hunk 2 above)
Comment (denisfruza) [reply WPFHospitalEditor/ScheduleWindow.xaml:27]: HAHAHAHAHAH

File: WPFHospitalEditor/Controller/ExaminationServerController.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+using WPFHospitalEditor.Controller.Interface;
+using WPFHospitalEditor.Service;
+using WPFHospitalEditor.Service.Interface;
+
+namespace WPFHospitalEditor.Controller
+{
+    public class ExaminationServerController : IExaminationServerController
+    {
+        IExaminationServerService examinationServerService = new ExaminationServerService();
+        public string ScheduleExamination(DateTime startTime, int doctorId, int patientId)
```
Comment (BoKna031) [inline WPFHospitalEditor/Controller/ExaminationServerController.cs:11]: What should the returned value represent? It is probably better to return the scheduled examination or null if the scheduling failed.

File: WPFHospitalEditor/ScheduleWindow.xaml (hunk 2 above)
Comment (denisfruza) [reply WPFHospitalEditor/ScheduleWindow.xaml:27]: fixed

File: WPFHospitalEditor/ScheduleWindow.xaml.cs (hunk 3 above)
Comment (denisfruza) [reply WPFHospitalEditor/ScheduleWindow.xaml.cs:49]: fixed

File: WPFHospitalEditor/Controller/Interface/IExaminationServerController.cs (hunk 4 above)
Comment (denisfruza) [reply WPFHospitalEditor/Controller/Interface/IExaminationServerController.cs:8]: fixed

File: WPFHospitalEditor/ScheduleWindow.xaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+            <Setter Property="Background" Value="SkyBlue"/>
+            <Setter Property="Foreground" Value="White"/>
+            <Setter Property="BorderThickness" Value="0"/>
+        </Style>
+    </Window.Resources>
+    <Grid>
+        <Border BorderThickness="10" BorderBrush="#FF8BD9E0" x:Name="Border">
+            <Grid x:Name="DynamicGrid" Background="#FFC6F5F8">
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Top" Margin="0,10,0,0">
+                    <ComboBox Name="patients" Width="200"/>
```
Comment (BoKna031) [inline WPFHospitalEditor/ScheduleWindow.xaml:23]: Using a combo box to find a patient is not a practical way!

**Review by rammba — APPROVED** [review]

File: WPFHospitalEditor/Controller/ExaminationServerController.cs (hunk 5 above)
Comment (denisfruza) [reply WPFHospitalEditor/Controller/ExaminationServerController.cs]: fixed

File: WPFHospitalEditor/ScheduleWindow.xaml (hunk 6 above)
Comment (denisfruza) [reply WPFHospitalEditor/ScheduleWindow.xaml:23]: Will be fixed in issue :)

**Review by BoKna031 — APPROVED** [review]

**Pull request merged**